tests/Backend/Snowflake/CloneIntoWorkspaceTest.php

```diff
@@ -303,20 +303,39 @@ public function testTableAlreadyExistsAndOverwrite()
         $workspaces = new Workspaces($this->_client);
         $workspace = $workspaces->createWorkspace();
 
+        $client2 = $this->getClientForToken(
+            STORAGE_API_LINKING_TOKEN
+        );
+
         $tableId = $this->createTableFromFile(
             $this->_client,
             $this->getTestBucketId(self::STAGE_IN),
             self::IMPORT_FILE_PATH
         );
 
-        $tableSecondId = $this->createTableFromFile(
-            $this->_client,
-            $this->getTestBucketId(self::STAGE_IN),
+        $this->dropBucketIfExists($this->_client, 'out.c-linked-bucket');
```

Comment:
bucket není parallel-safe, imho by to v novém kódu mělo už být vždycky parallel safe, ať si nepřidáváme práci